src/System.Globalization/tests/System.Globalization.Tests.csproj

```diff
@@ -144,5 +146,8 @@
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="CharUnicodeInfo\Unicode_8_0_0_CategoryData.txt" />
```

Comment:
It is better to add Logical name to this embedded resource and then use Assembly.GetManifestResourceStream(logicalName); so you don't have to enumerate all resources to get what you need